spec/support/ansible_shared/automation_manager/refresher.rb

```diff
@@ -100,12 +100,12 @@
   def assert_counts
     expect(Provider.count).to                                 eq(1)
     expect(automation_manager).to                             have_attributes(:api_version => "3.0.1")
-    expect(automation_manager.configured_systems.count).to    eq(130)
```

Comment:
yeah, was actually considering this. but thinking of using the `Faraday` direct call to avoid depending on the `Ansible Tower Gem`. What do you think? 
`automation_manager.provider.connect.hosts.count` would be much more straightforward.